docs/build.sbt

```diff
@@ -4,7 +4,7 @@ val AkkaVersion = "2.5.13"
 val JUnitVersion = "4.11"
 val JUnitInterfaceVersion = "0.11"
 val ScalaTestVersion = "3.0.4"
-val PlayVersion = "2.6.15"
+val PlayVersion = "2.7.0-M1"
```

Comment:
It would be good to check which other dependencies have inconsistent versions between docs and framework.